src/runTsc.js

```diff
@@ -42,17 +42,16 @@ const runTsc = ({ testPath, config: jestConfig }) => {
     process.cwd()
   );
 
-  const options = Object.assign({}, { noEmit: true }, settings.options);
-
-  const program = ts.createProgram([testPath], options);
-
-  const emitResult = program.emit();
+  const options = {
+    compilerOptions: {
+      noEmit: true,
+      ...settings.options,
+    },
+  };
 
-  const allDiagnostics = ts
-    .getPreEmitDiagnostics(program)
-    .concat(emitResult.diagnostics);
+  const transpileOutput = ts.transpileModule(testPath, options);
```

Comment:
Should the 1 argument be content?

Reply:
You're totally right. 

Also I'm proposing to **revert this PR** as it's actually not working correctly. @azz 

Not sure how to solve [the slowness](https://github.com/azz/jest-runner-tsc/issues/5), but @sibelius might be right  that we need a language server running in background to make it fast.

Reply:
on ts3.4 there is an incremental option that can make this even faster

check here https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html

Reply:
I tried to generate the buildinfo file for the `incremental` compiler option and it has no speed impact.

Note, the buildinfo file was generated only while running `tsc` from command line and without  `noEmit` option. I could not get it generated using Compiler API, ie. calling `createProgram` directly.

Seems like Language server is the only option now.

Reply:
it is worth opening an issue on typescript about this?

I think even without `noEmit` it should generate this files for incremental approach

Reply:
How about reading the contents of the file before passing to A`transpileModule`?

Reply:
I was trying that. Unfortunately, `result.diagnostics` are always an empty array.